nearcore/src/lib.rs

```diff
@@ -498,3 +499,88 @@ pub fn start_with_config_and_synchronization(
         arbiters: vec![client_arbiter_handle, arbiter.handle()],
     })
 }
+
+pub fn recompress_storage(home_dir: &Path, dst_dir: &Path) -> anyhow::Result<()> {
```

Comment:
What about extracting this to a separate file so lib.rs doesn't grow huge?

Reply:
The file isn’t actually that large and my intention with this function is that we’ll remove it in a couple of releases once everyone who needed to use it used it. All new users won’t need to perform the recompression.